pkg/v1/providers/kapp-controller-values/helpers.star

```diff
@@ -0,0 +1,47 @@
+load("@ytt:data", "data")
```

Comment:
It is unfortunate, but probably practical for now, that are we essentially copying a bunch of overlays from lib and 02_addons/kapp-controller
It would be good to provide some note that to help remind authors about this non DRY-ness (to improve the chance that any changes affecting kapp-controller deployment are made in both places)
Thinking either some comments in the overlay files, or perhaps a README.md in kapp-controller specific directories if we do not want the notes to be included in the files we deploy.